examples/angular/angular.json

```diff
@@ -25,7 +25,7 @@
             ],
             "styles": [
               "src/styles.css",
-              "node_modules/@stoplight/elements-dev-portal/styles/elements-core.min.css"
+              "node_modules/@stoplight/elements/styles/styles.min.css"
```

Comment:
Not sure that's the proper way. If people use dev-portal, they should install dev-portal styles, since they are more general.

Reply:
What do you mean by more general? Should they differ? Currently they are build from `elements-core` and copied to `elements` and `elements-dev-portal`

Reply:
I'm with @mpodlasin on this.

> Currently they are build from elements-core and copied to elements and elements-dev-portal

That's an implementation detail. It may not be true soon if we decide to add specific styles to dev-portal. 

Also people that only install dev portal shouldn't need to be aware that they have a `node_modules/@stoplight/elements` directory. In some weird cases they really may not have one, it's not guaranteed unless you depend on `elements` directly. (For example it's perfectly legal for a package manager to install it as `node_modules/@stoplight/elements-dev-portal/node_modules/@stoplight/elements`)

Reply:
But then shouldn't we include both styles, for elements and dev-portal? What is making dev-portal more generic than elements and not the other way around?

Reply:
In case the angular example is using both (which I think it is) then yes, we should include both CSS files

Reply:
Yeah, you are actually correct.

It should be both. 

I just don't want to people impression that when they use `elements-dev-portal`, they should use `elements` styles, that's all.